sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClientLiveTests.cs

```diff
@@ -166,16 +169,30 @@ public async Task HookOperations()
         {
             var adminClient = GetMetricsAdvisorAdministrationClient();
 
-            AlertingHook createdHook = await adminClient.CreateHookAsync(new EmailHook(Recording.GenerateAlphaNumericId("test"), new List<string> { "foo@contoso.com" })).ConfigureAwait(false);
-            AlertingHook getHook = await adminClient.GetHookAsync(createdHook.Id).ConfigureAwait(false);
-            List<AlertingHook> hooks = await adminClient.GetHooksAsync(new GetHooksOptions { HookName = getHook.Name }).ToEnumerableAsync().ConfigureAwait(false);
+            AlertingHook createdEmailHook = await adminClient.CreateHookAsync(new EmailHook(Recording.GenerateAlphaNumericId("test"), new List<string> { "foo@contoso.com" }) { Description = $"{nameof(EmailHook)} desscription" }).ConfigureAwait(false);
+
+            EmailHook getEmailHook = (await adminClient.GetHookAsync(createdEmailHook.Id).ConfigureAwait(false)).Value as EmailHook;
+
+            getEmailHook.Description = "updated description";
+            getEmailHook.EmailsToAlert.Add($"{Recording.GenerateAlphaNumericId("user")}@contoso.com");
+
+            await adminClient.UpdateHookAsync(getEmailHook.Id.ToString(), getEmailHook).ConfigureAwait(false);
```

Comment:
I think it would be worth adding the scenarios I mentioned for `DataFeed` here as well.

Reply:
Also, what about calling another `Get` and asserting that the value has been updated?